Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -67,67 +67,55 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
     internal var _drawUnitInChart = false
     
     /// The object representing the labels on the x-axis
-    internal var _xAxis: XAxis!
+    internal lazy var _xAxis: XAxis! = XAxis()
     
     /// The `Description` object of the chart.
     /// This should have been called just "description", but
-    @objc open var chartDescription: Description?
+    @objc open lazy var chartDescription = Description()
     
     /// This property is deprecated - Use `chartDescription.text` instead.
     @objc @available(*, deprecated: 1.0, message: "Use `chartDescription.text` instead.")
     open var descriptionText: String
     {
-        get { return chartDescription?.text ?? "" }
-        set { chartDescription?.text = newValue }
+        get { return chartDescription.text ?? "" }
```

Comment:
it seems we have a future conflict that these will be removed, but we change it now?
I'm not sure if we should merge https://github.com/danielgindi/Charts/pull/2996 into master now or for 4.0. I don't think it's worthy to refactor what will be removed

Reply:
It’s not a big deal though, and I will likely be the one handling merge conflicts. In this case, these deprecations need to be changed in order for the project to build.

I think we should merge into master, and I’ll handle any merge conflicts when we merge into 4.0 in the future.

Reply:
I have merged #2996 to make things easier

Reply:
Conflicts are resolved